crates/codegen/parser/runtime/src/napi_interface/cst.rs

```diff
@@ -50,11 +59,11 @@ impl RuleNode {
     }
 
     #[napi(ts_return_type = "Array<cst.Node>", catch_unwind)]
-    pub fn children(&self, env: Env) -> Vec<JsObject> {
+    pub fn children(&self) -> Vec<Either<RuleNode, TokenNode>> {
         self.0
             .children
```

Comment:
question unrelated to this PR, but I noticed that the NAPI `.children()` API here don't return the labels, opposite to the Rust API. Is this expected?